.travis.yml

```diff
@@ -1,13 +1,11 @@
 dist: xenial
-sudo: false
```

Comment:
Why remove `sudo: false`?

Reply:
It's no longer needed. Please see https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration